websites/U/uwowocosplay/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Uwowocosplay",
+	"description": {
+		"en": "Uwowo is a professional cosplay costume brand focusing on the high quality anime and game costumes and wigs with affordable price."
+	},
+	"url": "uwowocosplay.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/OczeyDO.jpeg",
```

Logo must be 512x512
```suggestion
	"logo": "https://i.imgur.com/U2EMMgC.jpeg",
```